zoom-meeting-sdk-linux/h/meeting_service_components/meeting_configuration_interface.h

```
@@ -1,1084 +1,1100 @@
-	virtual SDKError EnableZoomDocs(bool bEnable) = 0;
-#endif
-};
-
-END_ZOOM_SDK_NAMESPACE
+/**
+ * @file meeting_configuration_interface.h
+ * @brief Meeting Service Configuration Interface.
+ * @note Valid only for ZOOM style user interface mode.
+ */
+#ifndef _MEETING_Configuration_INTERFACE_H_
+#define _MEETING_Configuration_INTERFACE_H_
```

Clang (14.0.6)

[warning] declaration uses identifier '_MEETING_Configuration_INTERFACE_H_', which is a reserved identifier

(bugprone-reserved-identifier)